xrootd/authorization_test.go

```diff
@@ -157,4 +157,8 @@ func TestWriteOriginScitokensConfig(t *testing.T) {
 	require.NoError(t, err)
 
 	assert.Equal(t, string(monitoringOutput), string(genCfg))
+
+	configPath, err := ConfigXrootd(true)
```

Comment:
Now that we've revived the separate test for `ConfigXrootd`, can we remove these changes?

Reply:
Yeah. This was changed locally but didn't seem to have gotten pushed.